release.yaml

```diff
@@ -26,8 +30,10 @@ rancher-backup-crd:
 - 102.0.0+up3.1.0-rc1
 rancher-cis-benchmark:
 - 3.0.1-rc8
+- 4.0.0-rc1
 rancher-cis-benchmark-crd:
 - 3.0.1-rc8
```

Comment:
release.yaml should not have the 3.0.1-rc8 version, will we be removing this in a separate commit?

Reply:
I was kind of curious if we should have `3.0.1-rc8` regardless, since it contains the fix to the RBAC issue that prevented upgrades.

Reply:
`make remove`d as requested! ✔️